types/iva/defects/get/index.d.ts

```diff
@@ -10,14 +10,15 @@ export type InspectionType = "basic" | "normal";
 export interface SectionIVA {
```

Comment:
Why are we leaving the inspection type to be a type and not an enum? @m-coslett @cb-cs

Reply:
If you look at the conversation in the Romans DVSA chat this is discussed, I believe Michael originally had this as an enum, but when comparing to v3, the enum was only explicitly defined when it was a key/value pair. Anything with just values was defined ine the way it as above.

I guess it comes down to what you decided from your spike, and what the front-end are expecting / agreed to receieve.